### PR TITLE
bshultz-toast/DOC-1938

### PR DESCRIPTION
**Pull request by bshultz-toast**
Condensed summaries. Looks like someone already input operationIds.

### CODE REVIEW

**Comment by bshultz-toast** [comment]
-added `host: toast-api-server` at the top level of the spec
-added a 'contact' field with 'email' (odd little section issue? Had difficulty wrapping it cleanly after the description, there's little hash next to Line 19)
-reduced top-level `description` field to one sentence
-added/fixed `operationId`s
-edited summaries
-stripped HTML link from the one object mentioned

File: openapi/toast-stock-api.yaml [hunk 1]
```diff
@@ -4,20 +4,15 @@ info:
...
-      You can use the Toast Stock API to:
-
-      * Get all inventory information for a restaurant.
-      * Get the inventory for a list of menu items.
-      * Get the inventory of a restaurant given a specific status 
-        of the menu items (such as all menu items that are out
-        of stock).
+           
+    contact:
+    email: developer-support@toasttab.com
```
Comment (pdesjardins) [inline openapi/toast-stock-api.yaml:12]: I know this was in the spreadsheet, and I'm fine with merging this PR and updating the email in a different PR. But if the file is still open, would you remove the `email` line entirely and include only:

```
    contact: Toast developer support
```

File: openapi/toast-stock-api.yaml [hunk 2]
```diff
@@ -67,11 +62,11 @@ paths:
           description: Restaurant did not have any inventory information.
   /inventory/search:
     post:
-      summary: Returns inventory information for a list of menu items.
+      summary: Create inventory status for menu items
       description: |
         The inventory status of each menu item in the provided list will
         be returned.
-      operationId: searchInventoryByItemGuids
+      operationId: inventoryByItemGuidsPost
```
Comment (pdesjardins) [inline openapi/toast-stock-api.yaml:69]: Should this be `inventorySearchPost`?

I think the plan is to concatenate the exact path and the HTTP method.

File: openapi/toast-stock-api.yaml [hunk 3]
```diff
@@ -83,8 +78,7 @@ paths:
           type: string
         - name: body
           description: |
-            An <a
-            href="#/definitions/InventorySearchRequest">`InventorySearchRequest`</a>
+            An InventorySearchRequest
```
Comment (pdesjardins) [inline openapi/toast-stock-api.yaml:81]: Minor: we can give this code formatting by wrapping the object name in accent characters.

> An `InventorySearchRequest`.

File: openapi/toast-stock-api.yaml [hunk 4]
```diff
@@ -7,8 +7,9 @@ info:
   
       The Toast Stock API retrieves inventory information for
       menu items and modifier options from a restaurant.
-           
-    contact: Toast developer support
+      
+  contact: 
+    name: Toast developer support
```
Comment (bshultz-toast) [inline openapi/toast-stock-api.yaml:12]: When I initially edited the `contact` it added this odd hash mark in Geany on the line I couldn't figure out how to remove but it wouldn't pass a check with it. After adding `name` it seems to pass muster

Comment (jeanninecullen-toast) [reply openapi/toast-stock-api.yaml:12]: Looks good to me!

**Review by jeanninecullen-toast — APPROVED** [review]

**Review by pdesjardins — APPROVED** [review]
Looks clean and this PR is passing the build tests. Merge! 👍

**Pull request merged**